data/expected_match.csv

```diff
@@ -0,0 +1,7 @@
+email_address_field,1,3
+participant1@bhg.org,7,4
+participant2@bhg.org,6,2
+participant3@bhg.org,5,4
+participant4@bhg.org,3,0
+participant5@bhg.org,1,1
+participant6@bhg.org,6,3
```

Comment:
Do we really need this file? Without the ID the scores don't have much meaning, do they?

Reply:
This file is just an example of the expected result with the example data. The ID of the issue/project is the column name/first line.